flake.nix

```diff
@@ -67,6 +67,9 @@
             hardeningDisable = [ "fortify" ];
 
             shellHook = ''
+              # this CF CLI plugin is required by several make-targets
+              cf install-plugin -r CF-Community app-autoscaler-plugin -f
```

Comment:
This is not ideal since this approach downloads the plugin with every Nix devshell start. I am definitely open for other approaches here.

Reply:
Maybe the plugin could packaged the same as any other Go app and then installed from the filesystem?

Reply:
Awesome idea, implemented with 6a8f2b4. The shell starts much faster now 🚀